tree/ntuplelite/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+############################################################################
+# CMakeLists.txt file for building ROOT ntuple lite package
+# @author Oksana Shadura CERN
+############################################################################
+
+if(NOT root7)
+  return()
+endif()
+
+set(ROOTNTUPLELITE_SOURCES
```

Comment:
We need to specify the header file, too, don't we?

Reply:
For linker library, actually not...

Reply:
Anyway I want to land it in a separate PR, it was for Philippe to show how it will look like.